### PR TITLE
Guiding text added to select element in TestSelectForm.js

### PR DESCRIPTION
**Pull request by PR4NJ41**
Resolves Issue #738 

### Before
Screenshot from 2024-03-02 01-50-16

### After
Screenshot from 2024-03-02 01-48-27

### CODE REVIEW

**Comment by rahul6603** [comment]
Hi @PR4NJ41 , I have worked on Workplan by test in the PR https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/655 , and I suggest having consistent logic across all pages (similar work was done in PRs https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/656 and https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/665). I have proposed a simpler change in a reviewed comment.

File: frontend/src/components/workplan/TestSelectForm.js [hunk 1]
```diff
@@ -7,33 +7,30 @@ import { getFromOpenElisServer } from "../utils/Utils";
...
 
   useEffect(() => {
     mounted.current = true;
-    let testId = new URLSearchParams(window.location.search).get(
-      "testId"
-    );
-    testId = testId ? testId : ""; 
     getFromOpenElisServer("/rest/tests", (fetchedTests) => {
-      let test = fetchedTests.find(test => test.id === testId);
-      let testLabel = test ? test.value : "";
```
Comment (rahul6603) [inline frontend/src/components/workplan/TestSelectForm.js:31]: Can we just change the testLabel here to `testLabel = test ? test.value : "Select Test Type"`

**Review by CalebSLane — CHANGES_REQUESTED** [review]
Text should use the internationalization functionality available by using intl.formatMessage({}), providing a message key, and placing the english value in en.json. We also support French (fr.json), but that can be added independently of this commit.

**Comment by codewithunknown** [comment]
Amazing

**Comment by mozzy11** [comment]
> Text should use the internationalization functionality available by using intl.formatMessage({}), providing a message key, and placing the english value in en.json. We also support French (fr.json), but that can be added independently of this commit.

@PR4NJ41 can you resoleve the merge conflicts and also adrees the above comment ??

**Pull request merged**